### PR TITLE
Es6 based functionality in awesome books

### PR DESCRIPTION
**Pull request by Abdulwhbhussain**
Setup your project:

Create a new directory and initialize it with git.
Use npm init -y command to create package.json file.
Create the entry point for your JavaScript code called index.js
Create index.css file for your styles.
Create the main index.html file. Link your index.js and index.css files there.
Create a directory called modules.
Re-create your Awesome books app following the rules:

Use ES6 syntax.
Divide your code into modules.
Create a separate file for each module in the modules directory.
Import modules into the index.js file. (Tip: the more modules the better!)
Keep all funcionalities of the app without errors.
Refactor your functions to arrow functions.
Use let and const in the correct way.
Display the current date below the navigation bar.

For this requirement you should use a library [Luxon](https://moment.github.io/luxon/#/install?id=es6)
Download the library into your project using npm.
Import it as a module.
See the article [Get the current date and time](https://moment.github.io/luxon/#/tour?id=get-the-current-date-and-time). Your task here is to use a new tool effectively with little externally curated support. Exactly as you will do in your future job.

### CODE REVIEW

File: modules/sections.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+const contactSection = () => {
+  document.querySelector('#contact-link').style.textDecoration = 'underline';
+  document.querySelector('#contact-info').style.display = 'Block';
+  document.querySelector('#list-link').style.textDecoration = 'none';
+  document.querySelector('#add-link').style.textDecoration = 'none';
+  document.querySelector('.book-list-container').style.display = 'none';
+  document.querySelector('.book-form').style.display = 'none';
+};
+
+export { listSection, addSection, contactSection };
```
Comment (zieeco) [inline modules/sections.js:28]: - Hey, the navigation is not working. when clicked, it doesn't change to the different sections of your webpage. please, fix them.

File: modules/display-time.js [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+    completeDateAndTime += '0';
+  }
+  completeDateAndTime += DateTime.now().c.second;
+  completeDateAndTime += ' ';
+  completeDateAndTime += DateTime.now().c.hour > 12 ? 'pm' : 'am';
+
+  dateContainer.innerText = completeDateAndTime;
+};
+
+export default displayTime;
```
Comment (zieeco) [inline modules/display-time.js:43]: - Hey, I don't seem to see this date section on your webpage. kindly fix this issue.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+                <input type="text" id="title" placeholder="Book Title">
+                <br>
+                <input type="text" id="author" placeholder="Book Author">
+                <br>
+                <div id="button-div"><button type="submit" id="submit-btn">Add</button></div>
+            </form>
+        
+        
+        
+        </section>
```
Comment (zieeco) [inline index.html:48]: - Hey, after adding the book, one doesn't seem to see the added books. please, let's fix and improve the app.

**Review by zieeco — CHANGES_REQUESTED** [review]
Hi @Abdulwhbhussain,





Great job so far! 👏 
You've done well, there are some issues that you still need to work on to go to the next project, but you are almost there!


## To highlight 🎯

- No linter errors.✔️
- Nice design. ✔️
- Descriptive PR. ✔️


## Required Changes ♻️
_Check the comments under the review._




## Optional suggestions
- N/A

**_Every comment with the **[OPTIONAL]** prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._**


Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: modules/sections.js (hunk 1 above)
Comment (Abdulwhbhussain) [reply modules/sections.js:28]: @zieeco it is working 100% fine on My Laptop.

File: modules/display-time.js (hunk 2 above)
Comment (Abdulwhbhussain) [reply modules/display-time.js:43]: @zieeco it is working 100% fine on My Laptop.

File: index.html (hunk 3 above)
Comment (Abdulwhbhussain) [reply index.html:48]: @zieeco it is working 100% fine on My Laptop.

**Review by danmainah — APPROVED** [review]
Hi @Abdulwhbhussain , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🎉 🎉 

## Highlights
✔️ No linter errors
✔️ Github flow was correctly used
✔️  Good design matching framework given

## Optional suggestions

- [ ] @Abdulwhbhussain great work on your app, however, I noted that you repeatedly used the same commit message for your commits, it would be nice in the future to ensure every commit has a meaningful and unique commit message 
image


_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 3 above)
Comment (Abdulwhbhussain) [reply index.html:48]: It was Because of Luxon Dependency Library and Local Environment.

**Pull request merged**